### ISSUE TITLE
Is it working?

### ISSUE THREAD

**Issue by oOBlueArrowOo** [finndev/PokeBuddy#341]

**Comment by IMdrew** [comment]
yes!

**Comment by JackSbirrow** [comment]
How? It says "Failed to login", do i need to re-download it?

**Comment by oOBlueArrowOo** [comment]
@JackSbirrow maybe i did

**Comment by 404unknown** [comment]
@JackSbirrow Yes you need to redownload the whole zip and  overwrite everything except settings.json

**Comment by IMdrew** [comment]
you have to redownload it

**Comment by eswx13** [comment]
i'm trying it out now!

**Comment by eswx13** [comment]
just delete every other file u have. And then re-download it. should work

**Comment by 404unknown** [comment]
@eswx13 you should probably not delete your settings.json so you dont have to set everything up every update

**Comment by JackSbirrow** [comment]
It always says: [Error] GoogleAuthError: Object reference not set to an instance of an object.

**Comment by eswx13** [comment]
@404unknown yes i know. but i dont have any problem setting up everything again

**Comment by eswx13** [comment]
@JackSbirrow wait a sec

**Comment by eswx13** [comment]
@JackSbirrow https://github.com/finndev/PokeBuddy/commit/eadc091b0d4142f0a1f107aa62217f5cb58fd22f

-> go to this link .
-> search for a button called "view"
-> and then click in " View Raw"

it should start downloading the file. after that you just unzip it and start it like you did before

**Comment by JackSbirrow** [comment]
Still says it:
[P - 17:38:13] Attempting to Login...
[P - 17:38:13] Logging in with method: Google as Auth Provider
[P - 17:38:13] [Error] GoogleAuthError: Object reference not set to an instance of an object.
[P - 17:38:13] Failed to login.
[P - 17:38:13] Stopping...

**Comment by eswx13** [comment]
Did you mark the optiion google account on settings?

**Comment by JackSbirrow** [comment]
Yes

**Comment by eswx13** [comment]
ok thats weird. I don't know if servers are down for those who have google accounts.

**Comment by eswx13** [comment]
where are you from?

**Comment by JackSbirrow** [comment]
Italy

**Comment by eswx13** [comment]
i can't figure out! make sure you use the correct password and/or email

**Comment by JackSbirrow** [comment]
Are there anymore bots that work right now?

**Comment by eswx13** [comment]
Necro Bot i guess.

**Comment by JackSbirrow** [comment]
Ok i'm gonna try it

**Comment by eswx13** [comment]
Sorry if i couldn't help.

**Comment by JackSbirrow** [comment]
NecroBot error:
NecroBot-master\FeroxRev\PokemonGo.RocketAPI\PokemonGo.RocketAPI.csproj : error :impossible to load the project file.
This error is for all the projects that VisualStudio loads

**Comment by eswx13** [comment]
I don't know man. Create another post about pokebuddy, and wait for someone that can help you

**Comment by Enviouse** [comment]
Hey guys I'm sorta new to this but I waited for the new update have downloaded it, it works fine was getting the usual 40k exp per hr n cought 24 Pokemon when a error occurred catching Pokemon tells me somthing about nitanic can detect when u catch Pokemon in different locations hence won't allow me to catch any more and every encounter come with this msg anyone that can help
